devicesetup/src/main/res/layout-xlarge/activity_connecting.xml

```diff
@@ -0,0 +1,329 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Did this really need to be a whole separate layout?  Were some `dimens` values not sufficient?